app/uk/gov/hmrc/apisubscriptionfields/repository/FieldsDefinitionRepository.scala

```diff
@@ -59,24 +59,30 @@ class FieldsDefinitionMongoRepository @Inject()(mongoDbProvider: MongoDbProvider
         "apiContext" -> IndexType.Ascending,
         "apiVersion" -> IndexType.Ascending
       ),
-      indexName = Some("idIndex")
+      indexName = Some("apiContext-apiVersion_index"),
```

Comment:
I do not like using the word `id` 
We do not have ids anymore
Later we will need to change the code where we use the words `id` or `identifier`
It is confusing